.github/verify_header.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+
+#
+# Copyright(c) 2022 Intel Corporation
+# SPDX-License-Identifier: BSD-3-Clause
+#
+
+YEAR=$(date +"%Y")
+REGEX="Copyright\(c\) [0-9]{4}-([0-9]{4}) |Copyright\(c\) ([0-9]{4}) "
+
+while read -r line; do
+    if [[ "$line" =~ $REGEX ]]; then
+        echo ${BASH_REMATCH[0]}
+        if [[ $YEAR == ${BASH_REMATCH[1]} || $YEAR == ${BASH_REMATCH[2]} ]]; then
+            echo $1 have appropriate license header
+            exit 0
+        fi
+        echo $1 have wrong license header year
```

Comment:
\+ 'exit 1' after wrong header year was found

Reply:
But if there are for example few headers then it will exit after first wrong match. 
Without  this 'exit 1' it will test every header, so if someone from other company will add his own header it will test it.